litellm/utils.py

```diff
@@ -4035,7 +4035,10 @@ def openai_token_counter(
     """
     print_verbose(f"LiteLLM: Utils - Counting tokens for OpenAI model={model}")
     try:
-        encoding = tiktoken.encoding_for_model(model)
+        if "gpt-4o" in model:
+            encoding = tiktoken.get_encoding("o200k_base")
```

Comment:
This shouldnt be needed

Reply:
how else would we calculate tokens for gpt-4o vs. gpt3.5 turbo

i'm thinking about azure deployments here @jeromeroussin

Reply:
You're thinking of the case when the deployment name isn't "gpt-4o"? There is no guarantee it is going to contain "gpt-4o" either... I thought that is what base_model was for?

Reply:
> There is no guarantee it is going to contain "gpt-4o" either
yea that's what the next line is for 

>  I thought that is what base_model was for

yup that does help. the aim of the if-check is to provide a good default